src/test.js

```diff
@@ -381,6 +381,13 @@ Test.prototype = {
 			return true;
 		}
 
+		if ( config.moduleId.length >  0 ) {
+			moduleId = generateHash( this.module.name );
```

Comment:
Similar to @leobalter's comment:

> Just a minor concern: testId is registered on for each test, maybe we should register moduleId as well. This way makes it easier to identify the module's hash.

If we're only generating the `moduleId` when a module id is provided as a URL param, how do users know what a module's id is?